main.tf

```diff
@@ -388,6 +422,32 @@ resource "aws_route53_record" "atlantis" {
   }
 }
 
+################################################################################
+# EFS
+################################################################################
+
+resource "aws_efs_file_system" "efs" {
+  count = var.enable_ephemeral_storage ? 0 : 1
```

Comment:
Add a new-line separator after `count` and `for_each` in all resources and blocks.